util/chunk/chunk_test.go

```diff
@@ -48,7 +49,9 @@ var _ = check.Suite(&testChunkSuite{})
 
 type testChunkSuite struct{}
```

Comment:
I think you should remove `pingcap/check` fixture since they are what we want to replace with.

Reply:
The problem is that `column_test`, `list_test` also use `testChunkSuite`, so I kept it in this PR.

Reply:
so you migrate part of `util/chunk` pkg in this PR?

Reply:
Sorry I just see your description. Fine.

Reply:
Sorry I just see your description. Fine.